src/depends.py

```diff
@@ -311,13 +312,13 @@ def check_openssl():
     return False
 
 
-# TODO: The minimum versions of pythondialog and dialog need to be determined
+# ..todo:: The minimum versions of pythondialog and dialog need to be determined
 def check_curses():
     """Do curses dependency check.
 
     Here we are checking for curses if available or not with check
-    as interface requires the pythondialog\ package and the dialog
-    utility.
+    as interface requires the pythondialog <https://pypi.org/project/pythondialog>
+    package and the dialog utility.
```

Comment:
backticks are missing: https://www.sphinx-doc.org/en/1.8/usage/restructuredtext/basics.html#hyperlinks

Reply:
Oh. Yes!! 
Thank you :)

Reply:
missing underscore